### PR TITLE
LPS - 200057 - Set a limit on file size at asset library level

### PR DESCRIPTION
**Pull request by manuelRives**
https://liferay.atlassian.net/browse/LPS-200057

### CODE REVIEW

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java [hunk 1]
```diff
@@ -120,6 +120,14 @@ private void _updateDLSizeLimitConfiguration(
...
 		Map<String, String[]> parameterMap = actionRequest.getParameterMap();
 
+		Long fileMaxSize = 0L;
+
+		String[] inputSize = parameterMap.get("fileMaxSize");
+
+		if ((inputSize.length != 0) && Validator.isNotNull(inputSize[0])) {
+			fileMaxSize = GetterUtil.getLong(inputSize[0]);
+		}
+
```
Comment (adolfopa) [inline modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java:130]: Don't use wrapper types unless strictly necessary (Long vs long).

Also, can you use `MapUtil` to get the value and get rid of the `inputSize` variable? `MapUtil.getLong(..., "fileMaxSize", 0)`.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java (hunk 1 above)
Comment (manuelRives) [reply modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java]: Changed, I was trying to unify it with the rest of the method, but completely agree that it is a better solution.

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java [hunk 2]
```diff
@@ -120,6 +121,8 @@ private void _updateDLSizeLimitConfiguration(
 
 		Map<String, String[]> parameterMap = actionRequest.getParameterMap();
 
+		long fileMaxSize = MapUtil.getLong(parameterMap, "fileMaxSize", 0L);
+
```
Comment (adolfopa) [inline modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java:125]: You should move this up before the `mimeTypeSizeLimits` declaration (in general, variable declaration should follow the order in which they're used as parameters). Alternatively, and if the auto sf doesn't complain, you may inline the variable.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java (hunk 2 above)
Comment (manuelRives) [reply modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java]: Updated, And as you expected SF accepted the inlined variable.

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by SylviaLuan** [comment]
ci:test:stable

**Comment by SylviaLuan** [comment]
ci:test:stable

**Comment by adolfopa** [comment]
ci:forward:force

**Pull request closed without merging**